tests/UpgradeRule/PHP/fixtures/update-visibility.testfixture

```diff
@@ -0,0 +1,26 @@
+{
+}
+------
+<?php
+
+use SilverStripe\ORM\DataObject;
+
+class Animal extends DataObject
+{
+    public static $db = [
```

Comment:
Can we try beefing up this unit test a bit? Maybe add a mix of private, public, protected. Have both static and instance properties. Have classes that extend other classes, and see if the child classes pick up the new visibility.